calico_node/startup/startup.go

```diff
@@ -434,6 +447,28 @@ func autoDetectCIDRByReach(dest string, version int) *net.IPNet {
 	}
 }
 
+// autoDetectCIDRBySkipInterface auto-detects the first valid Network on the interfaces
+// matching the supplied interface regexes.
+func autoDetectCIDRBySkipInterface(ifaceRegexes []string, version int) *net.IPNet {
+	incl := []string{}
+	excl := []string{
```

Comment:
Should define this set as a constant since it's used in a couple of places, and we want to keep them in sync.